vcpkg.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": [
```

Comment:
These would be easier to review if they were lexicographically sorted.

Reply:
SDL2 is missing.

Reply:
Is libpng missing, or is that no longer a dependency?

Reply:
This update was from October when I first built Luanti. I was able to run devtest with this configuration, but I'm happy to add requirements if they've changed since October.

Reply:
Sorted the lines :)

Reply:
I have both `sdl2` and `libpng` in my own vcpkg.json. I recall having to add `sdl2` in order to get Luanti to compile. It was the case that `libpng` was a dependency for Irrlicht, and I assume that is still the case. It may be that one of the other dependencies installs it transitively.

Reply:
Ref #15336, it is a "works on my machine" solution, I'll keep investigating

Reply:
Added SDL2, libpng wasn't necessary though.

Reply:
Resolving as the original command was `vcpkg install zlib zstd curl[winssl] openal-soft libvorbis libogg libjpeg-turbo sqlite3 freetype luajit gmp jsoncpp gettext[tools] sdl2 --triplet x64-windows`, which doesn't include `libpng`